### ISSUE TITLE
Greater control over nose, edge, and tail shapes

### ISSUE THREAD

**Issue by splitn2** [mikemag/MonkeyCAM#12]
Keen to understand how much control is possible over nose and tail planshape. Modern designs rightly or wrongly tend to have more than just curves but instead more sections are blocky or blunted joined by curves, some of this is possibly more fashion but it also serves for nose lift and maybe sometimes less nose catchy designs too. Would having multiple control points enable us to pull parts of the radius and lock down various parts?

Would it be possible to overlay a curve in the background that started/finished at the end of the effective edge where it transitions into the nose, and also travels through / intersects the actual nose tip so you could see how your own tweaks compared to a more round "traditional" nose shape?

cheers
Rich

**Comment by RegMeasures** [comment]
Unless anyone is already addressing this I might try and have a go at adding an alternative tail shape to allow a flatter tail for touring skis. I like @mikemag's suggestion of a variable length flat section in the middle with a bezier at each end connecting to the edge.

I can think about other options as well maybe? If there's any that are a high priority some photos or suggestions about how you would like them specified would be great. @splitn2 - to have "multiple flat sections of definable length joined by Bezier" as you suggest above I guess you would need to define the angle and length of each, or perhaps the width and distance to the end of the board at each transition point? Could get quite complicated?

**Comment by splitn2** [comment]
I like this idea @RegMeasures and it may actually help me with my first few cnc'd splitboard builds until the splitboard specific issues are covered by MonkeyCAM. Thanks again.

**Comment by splitn2** [comment]
oh yeah, tip and tail eh @RegMeasures is that ok?

**Comment by mikemag** [comment]
@RegMeasures, go for it!

**Comment by RegMeasures** [comment]
#35 is a first attempt at adding flat ends connected by a bezier curve. Example image below shows quite a wide flat section used to create a classic flat ski tail.
![exampleflatskitail](https://cloud.githubusercontent.com/assets/17330572/17209319/96b42574-5510-11e6-888a-88f387492c24.JPG)
Are there any other end shapes that are a high priority?

**Comment by splitn2** [comment]
legend! keep it coming @RegMeasures this is awesome mate. cheers

**Comment by splitn2** [comment]
if you really wanna get fancy a skin clip rebate to suit the G3 and BD tail clips? we'd need to get hold of some to measure up.

**Comment by splitn2** [comment]
@RegMeasures I suggest @sou-l might have a few good ideas on this he is well dialed in to trending board design and shapes....... Hes local too good fella.

**Comment by sou-l** [comment]
Can we get a reversed bezier where the flat section is to create a fish
tail?

Cheers Liam

**Comment by splitn2** [comment]
Good Idea @sou-l as well are primo pow board that would also double as a skin clip rebate on a touring ski maybe eh

In the mean time I think @NickAlbers is working on some splitboard MonkeyCAM code this week fingers crossed he's successful I want to get all your guys talking together more fun more shared knowledge more power to you all!

@RegMeasures thanks for the enthusiasm yes keen for catching up soon mate when ever is convenient. cheers

**Comment by mikemag** [comment]
With pull req https://github.com/mikemag/MonkeyCAM/pull/35 we now have flat end options. Thank you @RegMeasures!!

We'll keep this issue here since we'll want more.

**Comment by mikemag** [comment]
Flat tails in v4.0.5: https://github.com/mikemag/MonkeyCAM/releases/tag/4.0.5

**Comment by splitn2** [comment]
Awesome to have the new release @mikemag thanks for keeping an eye out for this and releasing 4.0.5 so fast!!!!! Thanks @RegMeasures for your coding effort, I am looking forward to building these skis for you , Partial Edges and Flat Tail are both great enhancements.

I'll follow up with @NickAlbers and see how he is getting on with the splitboard specific mods next.

@sou-l check the latest release!!!!!

cheers
Rich
